### PR TITLE
Feature Voice messages

### PR DESCRIPTION
**Pull request by nsklyarov**
# Contribution checklist

## Brief description

This pull-request added experimental functionality for recording and sending voice messages in the text editor.

**Demo**

https://github.com/hcengineering/anticrm/assets/12935413/cfb0fb53-bb82-4d0e-bca0-56c7bf4a66f0

## Checklist

* [ ] - UI test added to added/changed functionality?
* [ ] - Screenshot is added to PR if applicable ?
* [x] - Does the code work? Check whether function and logic are correct.
* [ ] - Does Changelog.md is updated with changes?
* [ ] - Does the translations are up to date?
* [ ] - Does it well tested?
* [x] - Tested for Chrome.
* [x] - Tested for Safari.
* [ ] - Go through the changed code looking for typos, TODOs, commented LOCs, debugging pieces of code, etc.
* [ ] - Rebase your branch onto master and upstream branch
* [ ] - Is there any redundant or duplicate code?
* [ ] - Are required links are linked to PR?
* [ ] - Does new code is well documented ?

---------

### Major changes:

- Added controls for voice recording.
- Implemented integration with audio recorder for recording and sending voice messages.
- Added a new `useVoiceControls` prop that allows you to display or hide voice recording controls.

### How it works:

1. Users can activate voice recording mode by hold mouse click on the appropriate button.
2. When voice recording is activated, the audio recorder starts recording.
3. After the recording is complete, users can send a voice message by pressing the send button.

### CODE REVIEW

File: packages/ui/src/index.ts [hunk 1]
```diff
@@ -207,7 +208,7 @@ export * from './colors'
 export * from './focus'
 export * from './resize'
 
-export function createApp (target: HTMLElement): SvelteComponent {
```
Comment (haiodo) [inline packages/ui/src/index.ts]: Please call rush format, before commit changes. Since linter will not pass a build.

**Comment by haiodo** [comment]
@nsklyarov Nikita could you please also commit with -s -S to have a signed commits and GPG signed commits, we have a rule for all contributions. Thanks.

File: packages/ui/src/index.ts (hunk 1 above)
Comment (nsklyarov) [reply packages/ui/src/index.ts]: Ok, did

**Comment by nsklyarov** [comment]
> @nsklyarov Nikita could you please also commit with -s -S to have a signed commits and GPG signed commits, we have a rule for all contributions. Thanks.

Yeah, I signed everything. DCO check cleared.

File: packages/text-editor/src/AudioRecorder.ts [hunk 2]
```diff
@@ -0,0 +1,57 @@
+export default class AudioRecorder {
```
Comment (haiodo) [inline packages/text-editor/src/AudioRecorder.ts:16]: @nsklyarov 
Could you please add Header with License. Thanks.

Comment (nsklyarov) [reply packages/text-editor/src/AudioRecorder.ts:16]: Sure! Added!

**Pull request closed without merging**